### DISCUSSION TITLE
Responsive UI

### DISCUSSION THREAD

**Discussion by AthulA10** [fastn-stack/fastn#762] [Q&A] [answered]
Can you provide a reference static page code for knowing how to make the page responsive?

**Comment by amitu** [comment, marked as answer]
To make a site responsive you have two techniques. One is to use `if` conditions explicitly ([example](https://github.com/FifthTry/doc-site/blob/main/index.ftd#L277)). A better approach is to use our built in support for [responsiveness for font](https://fastn.io/ftd/built-in-types/#ftd-responsive-type),  and [length](https://fastn.io/ftd/built-in-types/#ftd-length). 

Later approach avoid explicit if condition so your code is slightly leaner, also for built in stuff we can do more optimisations going forward.
